lib/src/primitives/check_duplicates.dart

```diff
@@ -0,0 +1,36 @@
+import 'dart:convert';
+
+/// Check for duplicated values in arb file string.
+/// return a Map containing duplicated key-value pairs.
+Map<String, dynamic> checkDuplicatesARB(String arbContent) {
+  final Map<String, dynamic> arbJsonMap = json.decode(arbContent);
+  arbJsonMap.removeWhere((key, value) => key.startsWith('@@'));
+  arbJsonMap.removeWhere((key, value) => key.startsWith('@'));
+  arbJsonMap.removeWhere((key, value) {
```

Comment:
what's the intention of this code? write it down on a //comment

Are you sure the regexp works for every fomatting of the .arb? You'll have to format it before processing or improve the RegExp, maybe with \s\S